### DISCUSSION TITLE
How to use JWT with secrets via Cloudflare

### DISCUSSION THREAD

**Discussion by yusukebe** [honojs/hono#801] [Q&A] [answered]
How to use JWT with secrets via Cloudflare?

This question is originated by #756

**Comment by yusukebe** [comment, marked as answer]
We can write the following way:

```ts
type Bindings = {
  JWT_TOKEN_KEY: string
}

const app = new Hono<{ Bindings: Bindings }>()
app.use('/*', async (c, next) => {
  const jwtMiddleware = jwt({
    secret: c.env.JWT_TOKEN_KEY,
  })
  return jwtMiddleware(c, next)
})
```

**Comment by Tholkappiar** [comment]
How to access wrangler secrets in Hono and also how to access cloudflare worker Variables and Secrets and use that in Hono when deploying ?

Thanks in advance !
